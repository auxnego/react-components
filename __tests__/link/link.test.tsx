import React from "react";
import {create} from "react-test-renderer";

import Link from "../../link/link";

it("normal link", () => {

  expect(create(
    <Link href="#active-link">actived Link</Link>
  ).toJSON()).toMatchSnapshot();

});

it("disabled link", () => {

  expect(create(
    <Link disabled>Disabled Link</Link>
  ).toJSON()).toMatchSnapshot();

});
