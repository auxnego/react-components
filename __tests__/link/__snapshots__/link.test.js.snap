// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`disabled link 1`] = `
<span
  className="link link--disabled"
  disabled={true}
>
  Disabled Link
</span>
`;

exports[`normal link 1`] = `
<a
  className="link"
  disabled={false}
  href="#active-link"
>
  actived Link
</a>
`;
